src/state/Context.tsx

```diff
@@ -11,4 +11,15 @@ const cyclesInit: CyclesContextI = {
   updateCycles: (_newCycles) => {},
 };
 
+interface ThemeContextI {
```

Comment:
Usually interface name started with I, not ended

Reply:
Fixed